docs/quick_start/hands_on.md

```diff
@@ -70,39 +68,38 @@ Gamma
 4 4 4 0 0 0
 ```
 
-Ok, now that all key input files have been set, we should be able to run the first quick example. The simplest way is to use the command line, e.g.:
+After all the above input files have been set, one should be able to run the first quick example. The simplest way is to use the command line, e.g.:
 
 ```
-mpirun -np 4 abacus
+mpirun -np 2 abacus
 ```
 
-The main output information appears in `OUT.MgO/running_scf.log`, which starts with
+The main output information is stored in the file `OUT.MgO/running_scf.log`, which starts with
 
 ```
```

Comment:
Showing the content of output files does not provides useful info here, and becomes quite verbose. I would suggest removing them.

Reply:
It confirms users some sense of success, so that they are useful from my point of view.